src/components/basic/display/BracketsView/Footer.tsx

```diff
@@ -51,7 +51,7 @@ export const Footer = memo(function Footer(): JSX.Element {
         startPrice && (
           <PriceDisplay
             price={startPrice}
-            token={baseTokenAddress}
+            token={quoteTokenAddress}
```

Comment:
Maybe the `PriceDisplay` should name the property `quoteToken` to save future confusion?
```suggestion
            quoteToken={quoteTokenAddress}
```

Reply:
Good idea, changed